lib/ansible/config/base.yml

```diff
@@ -91,16 +91,15 @@ ANSIBLE_PIPELINING:
     - "However this conflicts with privilege escalation (become). For example, when using 'sudo:' operations you must first
       disable 'requiretty' in /etc/sudoers on all managed hosts, which is why it is disabled by default."
     - This option is disabled if ``ANSIBLE_KEEP_REMOTE_FILES`` is enabled.
+    - This is a global option, each connection plugin can override either by having more specific options or not supporting pipelining at all.
   env:
     - name: ANSIBLE_PIPELINING
-    - name: ANSIBLE_SSH_PIPELINING
   ini:
-  - section: connection
+  - section: defaults
     key: pipelining
-  - section: ssh_connection
```

Comment:
Maybe `ssh_connection` should stay in the middle for 2.10, just in case someone had the setting specified there and working? I'm +1 for all the rest of the changes here for 2.10, but removing that ability in a bugfix patch might break some folks.

Reply:
@nitzmahone the same entries work in the plugin itself, this is not used so its just redundant

Reply:
Right, but a previous `pipelining` entry under `ssh_connection` would have enabled it globally (including for `local`), yeah? So changing that behavior in a backport patch is probably not the right thing...

Reply:
it still does for plugins that declare it, like paramiko, for most others it doesn't matter

Reply:
... and that makes total sense going forward, but removing the value in a backport is potentially introducing breaking behavior, because the existing behavior in those versions was global, not per-plugin. It's a corner case, but also one that's easily rectified by just leaving `ssh_connection` in the middle of the global mix for 2.10.

Reply:
i had checked back into even 2.9, no plugin used this directly